packages/pn-pa-webapp/src/navigation/routes.tsx

```diff
@@ -15,9 +15,9 @@ function Router() {
     <Routes>
       <Route path="/" element={<VerifyUser />}>
         {/* protected routes */}
-        <Route
+        <Route path="/" 
```

Comment:
Cosa cambia tra prima (senza path) e adesso (con path)?

Reply:
Aggiungo che nella documentazione di React Router non ho trovato riferimenti al comportamento de un `Route` senza `path`, io evito di utilizzare funzionalità non documentate in librerie dentro del possibile. Prima di aggiungere il `path` alla `Route` di `RequireAuth`, ho incontrato qualche comportamento diverso da quello che mi aspettavo, purtroppo non ho scritto i dettagli, ed adesso andare indietro e provare le diverse casistiche mi porterebbe una quantità di tempo non triviale.  
Ho descritto in dettaglio il modo in cui interagiscono `VerifyUser` e `RequiredAuth` nel commentone in inglese, sezione "How the authentication is handled", dentro di "Some technical details" c'è una parte in merito.